fvm/transactionFeeDeductor.go

```diff
@@ -37,14 +36,24 @@ func (d *TransactionFeeDeductor) Process(
 func (d *TransactionFeeDeductor) deductFees(
 	vm *VirtualMachine,
 	ctx *Context,
-	tx *flow.TransactionBody,
+	proc *TransactionProcedure,
```

Comment:
Hmm, not sure if this is right change. Isn't `TransactionProcedure` supposed to be used only once

Reply:
yeah, this is a temp measure to be able to return the events back to the original caller through this proc, Fees are going to be refactored soon by Janez, so basically this should suffice to fix the current issue.